crates/cli/src/utils/cmd.rs

```diff
@@ -446,22 +448,31 @@ pub async fn print_traces(
     result: &mut TraceResult,
     decoder: &CallTraceDecoder,
     verbose: bool,
+    json: bool,
+    state_changes: bool,
 ) -> Result<()> {
     let traces = result.traces.as_mut().expect("No traces found");
 
-    sh_println!("Traces:")?;
+    if !json {
+        sh_println!("Traces:")?;
+    }
+
     for (_, arena) in traces {
         decode_trace_arena(arena, decoder).await?;
-        sh_println!("{}", render_trace_arena_with_bytecodes(arena, verbose))?;
+        sh_println!("{}", render_trace_arena_inner(arena, verbose, json, state_changes))?;
```

Comment:
If the user is using `-j` for json output, maybe it's expected to print the output regardless of the shell verbosity settings.

Reply:
when --json is passed some level of JSON output is to be expected where the verbosity flag `-v`, etc.. control to what degree the output is detailed

Reply:
does this mean in order to get the json output, we will need to pass `-v` in additonal to `-j`? I was thinking of keeping `println!` here, but it seems it has been add to `disallowed_macros` list.